package.json

```diff
@@ -6,7 +6,8 @@
     "build": "tsc",
     "test": "mocha test/**/*.spec.ts",
     "cover": "nyc mocha test/**/*.spec.ts",
-    "lint": "tslint -c tslint.json --project tsconfig.json",
+    "lint": "eslint --ext .ts src/ test/",
+    "lint:fix": "eslint --ext .ts src/ test/ --fix",
```

Comment:
Maybe we could add a new `format` npm script to run `prettier`.

Also instead to use `--ext` flag, since it's repetitive to have it twice with the same patterns, we should have it in the `.eslintrc.js` config file.
See: <https://eslint.org/docs/user-guide/configuring/configuration-files#configuration-based-on-glob-patterns>

Reply:
prettier is used as eslint's plugin, so it run together, no need for additional command.

Reply:
I've just been looking into this, it seems that config file options for glob patterns are only for overrides, not for top level execution... or am I missing something?